ArraysAndLists/Program.cs

```diff
@@ -18,50 +19,97 @@ public static class ArraysAndLists
         // If two people like your post, it returns: "[Friend 1] and [Friend 2] liked your post."
         // If more than two people like your post, it returns: "[Friend 1], [Friend 2] and [Number of Other People] others liked your post."
         public static string FacebookLikes(params string[] names)
+        
         {
-            // Delete the below line when you are writing your code, don't forget to use your
-            // own return statement.
-            return default;
+
+            if (names.Length == 0)
+                return null;
+            else if (names.Length == 1)
+                return $"{names[0]} liked your post.";
+            else if (names.Length == 2)
+                return $"{names[0]} and {names[1]} liked your post.";
+            else
+                return $"{ names[0]}, {names[1]} and {(names.Length - 2)} others liked your post.";
+
```

Comment:
This is a good way of solving this, the only change I would make is reducing the use of return by creating and storing the desired string in a string variable then returning that string variable at the end. Doing this reduces the total number of times the return function is used decreasing the chances of accidentally being kicked out of the method.